types/babel__traverse/index.d.ts

```diff
@@ -484,12 +619,20 @@ export class NodePath<T = Node> {
     getSource(): string;
 
     /** Check if the current path will maybe execute before another path */
-    willIMaybeExecuteBefore(path: NodePath): boolean;
+    willIMaybeExecuteBefore(target: NodePath): boolean;
+
+    resolve(dangerous?: boolean, resolved?: NodePath[]): NodePath;
+
+    isConstantExpression(): boolean;
+
+    isInStrictMode(): boolean;
     //#endregion
 
     //#region ------------------------- context -------------------------
     call(key: string): boolean;
 
+    isDenylisted(): boolean;
+
     isBlacklisted(): boolean;
```

Comment:
Should this be also removed here? I see you removed `blacklist` from the options too.

https://github.com/babel/babel/blob/main/packages/babel-traverse/src/path/context.ts#L63